dmn_meetstation/include/board.h

```diff
@@ -25,6 +25,23 @@
 extern "C" {
 #endif
 
+/**
+ * @name    Sensor configuration
```

Comment:
Maybe call this `CCS811 eCO2 sensor configuration`

Reply:
I've used 'gas sensor' as per https://ams.com/ccs811